### ISSUE TITLE
Update version of Chai.js and is-ip

### ISSUE THREAD

**Issue by carl-parrish** [BigstickCarpet/postman-bdd#10]
https://david-dm.org/BigstickCarpet/postman-bdd

Looks like these dependencies are out of date. (I just wasted a day trying to chase down a "bug" that was just I was looking at the update docs while using the outdated lib) Any release dates on when it will have the new version? Are there issues with the new version that other's can help resolve?

**Comment by carl-parrish** [comment]
I just attempted to build it myself with Chai 4.0.2 I only got one failed test.  

```
✖ should not throw
    -------------------
      operator: throws
      expected: |-
        undefined
      actual: |-
        { actual: '35', expected: undefined, message: 'expected \'35\' to be a number', showDiff: false }
      at: Test.test (/Users/cparrish/Desktop/Sandbox/postman-bdd/test/specs/response/body.spec.js:55:5)




  Failed Tests: There was 1 failure

    XML response body

      ✖ should not throw


  total:     951
  passing:   950
  failing:   1
  duration:  539ms
```

**Comment by BigstickCarpet** [comment]
Chai.js 4.0 just released last week, and it includes [many breaking changes](https://github.com/chaijs/chai/releases/tag/4.0.0), so I haven't had a chance to figure out a clean upgrade path for Postman BDD.  I can't just update the Chai dependency, because that will break many people's existing tests that use Postman BDD.  So I need to think about the best way to expose a backward-compatible Chai 3.x version of Postman BDD _and_ a new Chai 4.x version as well.

In the meantime, you should be able to use the existing version of Postman BDD, as long as you aren't relying on any of the new or changed functionality in Chai 4.0.

**Comment by BigstickCarpet** [comment]
@carl-parrish - I've merged your PR and released Postman BDD v5.0.0 with Chai 4.0.2.  I solved the backward-compatibility problem by adding version-specific subfolders under the `dist` folder (e.g. `dist/v3`, `dist/v4`, `dist/v5`)
